BEIMA.Backend.Test/DeviceFunctions/UpdateDeviceTest.cs

```diff
@@ -24,20 +25,29 @@ public async Task IdNotInDatabase_UpdateDevice_ReturnsNotFound()
             // Setup mock database client.
             Mock<IMongoConnector> mockDb = new Mock<IMongoConnector>();
             var testId = "abcdef123456789012345678";
-            mockDb.Setup(mock => mock.UpdateDevice(It.Is<BsonDocument>(bd => bd["_id"].AsObjectId.ToString().Equals(testId))))
```

Comment:
We still need the mock method for UpdateDevice no?

Reply:
Actually, it doesn't even look like we call GetDevice in the update endpoint, I am confused?

Reply:
I guess we technically don't actually need it since we are asserting that it is run Times.Never below. I think I'm ok with this actually.